### DISCUSSION TITLE
/docs 中使用的css/js cdn 访问不到

### DISCUSSION THREAD

**Discussion by zy7y** [Django-Simple-API/django-simple-api#58] [Q&A] [answered]
看了下fastapi也是用的这cdn 本地试了下 访问不到/docs,  目前是可配置的么,
![image](https://user-images.githubusercontent.com/67154681/170626015-8b045f28-f99e-4224-a8af-dba403a2a47e.png)

**Comment by waitan2018** [comment]
我也访问不到了，如果再给我一次机会，我一定把资源放本地一份。

**Comment by waitan2018** [comment, marked as answer]
突然又看到了这个问题，现在最新的版本中，静态资源都放在了本地，不走CDN了。。。
